web/src/features/panels/ranking-panel/getRankingPanelData.ts

```diff
@@ -33,15 +33,21 @@ export const getRankedState = (
   if (!data) {
     return [];
   }
-  const zonesData = data.data;
-  const keys = Object.keys(zonesData.zones) as Array<keyof GridState>;
+  const gird = data.data.datetimes[datetimeIndex];
```

Comment:
Is this supposed to be `grid`?

Reply:
I think so, as in grid state.

But I'll double check 🙂